connectors-home/modules/ROOT/nav.adoc

```diff
@@ -5,7 +5,6 @@
 * xref:introduction/intro-use-exchange.adoc[Using Exchange to Discover Connectors, Templates, and Examples]
 * xref:introduction/intro-connector-configuration-overview.adoc[Anypoint Connector Configuration]
  ** xref:introduction/intro-config-use-studio.adoc[Using Anypoint Studio to Configure a Connector]
```

Comment:
add page alias from this deleted Flow Designer page to the Studio page? Or even the Anypoint Connector Configuration page?